tools/esp8266/hmRadio.h

```diff
@@ -165,31 +165,26 @@ class HmRadio {
 
         void sendControlPacket(uint64_t invId, uint8_t cmd, uint16_t *data) {
             DPRINTLN(DBG_INFO, F("sendControlPacket cmd: ") + String(cmd));
-            sendCmdPacket(invId, TX_REQ_DEVCONTROL, SINGLE_FRAME, false); // SINGLE_FRAME 0x81 as original DTU code
```

Comment:
You are undoing earlier changes made for ClearState_LockAndAlarm and potentially other DevControl commands which do not have userdata. Maybe @ homeautomation22 better double checks his changes against the current dev03 branch ?

Reply:
Ist so richtig, ist wie vorher nur cnt jetzt verständlicher, CleanState_LockAndAlarm würde damit auch funktionieren, wenn es nicht doch noch 2 Datenbytes enthält(diese wären aber derzeit unbekannt). Kann man so damit senden, genau wie auch On/Off/Restart, kommt aber zumindest bei mir 0x04 zurück, also verweigert.